src/samples/http/server-sse.c

```diff
@@ -72,14 +72,28 @@ static struct sol_ptr_vector responses = SOL_PTR_VECTOR_INIT;
 static int port = 8080;
 static bool should_quit = false;
 
+static struct sol_blob sse_prefix = {
+    .type = &SOL_BLOB_TYPE_NO_FREE,
+    .parent = NULL,
+    .mem = (void *)"data: ",
+    .size = sizeof("data: ") - 1,
+    .refcnt = 1
+};
+
+static struct sol_blob sse_suffix = {
+    .type = &SOL_BLOB_TYPE_NO_FREE,
+    .parent = NULL,
+    .mem = (void *)"\n\n",
+    .size = sizeof("\n\n") - 1,
+    .refcnt = 1
+};
+
```

Comment:
I saw this code some times on this patch... couldn't it be Soletta responsibility to deal with these prefix and suffix stuff?